### PR TITLE
8.0 updates

### PR DESCRIPTION
**Pull request by alviteri**

### CODE REVIEW

**Comment by jerpelea** [comment]
please remove 379d444
v6 is comming soon

**Comment by jerpelea** [comment]
@AdFad666 review please

File: system_server.te [hunk 1]
```diff
@@ -15,3 +15,8 @@ dontaudit system_server hal_audio_default:file write;
 dontaudit system_server appdomain:file write;
 
 allow system_server proc_cmdline:file r_file_perms;
+allow system_server netmgrd_socket:dir search;
+allow system_server netmgrd_socket:sock_file write;
+allow system_server netmgrd:unix_stream_socket connectto;
```
Comment (AdrianDC) [inline system_server.te]: Use unix_socket_connect(system_server, netmgrd, netmgrd) instead of these two previous lines.

**Comment by AdrianDC** [comment]
@AdFad666 
unix_socket_connect already does "allow $1 $2_socket:sock_file write;", and that's all he added + shown in the denials log.

Detail about a commit : sepolicy: fix identation -> indentation.

**Comment by AdrianDC** [comment]
-2 on thermanager dac_override.

Most of the dac_overrides should be fixed by https://github.com/sonyxperiadev/thermanager/commit/c82490ef68fc5c9893e0dcb56fa6ad8f2ed47383,
and the last ones should be properly identified with an strace to find the missing service groups / sysfs permissions & groups overrides, and possibly unnecessary nodes.

For reference on my Huashan Oreo sepolicies :
https://github.com/LineageOS/android_device_sony_huashan/commit/3c1c6f88c00034dedc49c6dfdcc3b93473d8d1e3
https://github.com/LineageOS/android_device_sony_huashan/commit/01fbd289685339dfba1f0b4d691067863b3b406d
https://github.com/LineageOS/android_device_sony_huashan/commit/8afa18ca41f50c24cf3037aa5ea7b984c3d35a26

**Comment by alviteri** [comment]
sorry last one should not be on this branch

**Comment by AdFad666** [comment]
@AdrianDC good spot, I deferred to qcom sepolicy after I didn't find anything related to this in the google devices. qcom has the `netmgr_socket(clientdomain)` macro which defines exactly the above.

looking for something else I just found:
```
# TODO(b/36613917): Remove this once system_server no longer communicates with netmgrd over sockets.
typeattribute netmgrd socket_between_core_and_vendor_violators;
allow system_server netmgrd_socket:dir search;
unix_socket_connect(system_server, netmgrd, netmgrd)
```
in the marlin sepolicy, probably I checked an older branch the first time round.

**Pull request merged**